ranking/opts.go

```diff
@@ -57,10 +57,14 @@ func boolFromQuery(query url.Values, name string) bool {
 }
 
 func addSuffixesForFiletype(suffixes *map[string]float32, filetype string) {
+	// what exactly happens here? what does the number assign do?
+	// i'm just trying to add support for Objective-C source files
 	switch filetype {
 	case "c":
 		(*suffixes)[".c"] = 0.75
 		(*suffixes)[".h"] = 0.75
+	case "m":
```

Comment:
This should read “objc” instead of “m”. Also, please add the new file type to static/faq.html